osu.sln.DotSettings

```diff
@@ -209,6 +209,8 @@
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=StructCanBeMadeReadOnly/@EntryIndexedValue">WARNING</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuggestVarOrType_005FBuiltInTypes/@EntryIndexedValue">SUGGESTION</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuggestVarOrType_005FSimpleTypes/@EntryIndexedValue">DO_NOT_SHOW</s:String>
+	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuspiciousTypeConversion_002EGlobal/@EntryIndexedValue"></s:String>
+	<s:Boolean x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuspiciousTypeConversion_002EGlobal/@EntryIndexRemoved">True</s:Boolean>
```

Comment:
Does this remove the inspection? I thought we decided on having it error/warning instead?

Reply:
That's on the base PR, but it does look to correctly warn at my end:

![20211115 200404 (JetBrains Rider-EAP)](https://user-images.githubusercontent.com/191335/141771307-30b029f9-89ae-49d8-85bb-7a396e423ac4.png)

on master:
![20211115 200404 (JetBrains Rider-EAP)](https://user-images.githubusercontent.com/191335/141771427-76014305-afb6-45bd-893d-6c32199d120b.png)

Reply:
Yep, confirmed that it works. :psyduck: